aiogram/utils/helper.py

```diff
@@ -233,3 +235,60 @@ def all(cls) -> List[str]:
             else:
                 result.append(value)
         return result
+
+
+class DefaultProperty(Generic[T]):
```

Comment:
What do you think about renaming to `Default`?

Reply:
It's a property-like descriptor and intended for in-class only use, so I think `Property` suffix clarifies where does `DefaultProperty` belong.

On the other hand, we declare it as an internal feature -- sure we can rename it + adding nice docstring :)